src/main/scala/SBTS3Resolver.scala

```diff
@@ -85,7 +87,7 @@ object SbtS3Resolver extends AutoPlugin {
     implicit def fromProviderToAWSRegion(provider: AwsRegionProvider): Region = regionFromString(provider.getRegion())
 
     // Adding setting keys
-    lazy val awsProfile     = settingKey[String]("AWS credentials profile")
+    lazy val awsProfile     = settingKey[Option[String]]("AWS credentials profile")
```

Comment:
Regarding your concern about breaking change: take a look at the trick we did in https://github.com/ohnosequences/sbt-s3-resolver/pull/55#discussion_r140916416. You can add an implicit conversion from `String` to `Option[String]` and a deprecation message, which explains that you should write explicit `Some(...)`. This way users that didn't use the setting, won't notice the change and those who did will get a warning.

Reply:
I added an implicit conversion from `String` to `Option[String]` according to `acl2Option`